### ISSUE TITLE
Questions about split sets of SIM10K dataset

### ISSUE THREAD

**Issue by qianyuzqy** [thuml/Decoupled-Adaptation-for-Cross-Domain-Object-Detection#1]
Hello, thanks for your excellent paper and code! I want to follow your great work but got some problems.
I download the image set of SIM10K dataset from the official website and found the split is the combination of both the training set and the validation set. I want to know how the authors split the official set into a training set and a validation set?

**Comment by JunguangJiang** [comment]
Sorry for the late reply.
To be consistent with previous work, _trainval10k_ is used both in training and validation (Note that no annotations of Sim10k are used during training).
